symphony-bdk-legacy/symphony-api-client-java/src/test/java/services/DatafeedEventsServiceV1Test.java

```diff
@@ -0,0 +1,61 @@
+package services;
+
+import clients.SymBotClient;
+import clients.symphony.api.DatafeedClient;
+import configuration.SymConfig;
+import lombok.SneakyThrows;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+
+import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.atLeastOnce;
+
+public class DatafeedEventsServiceV1Test {
+    private SymBotClient symBotClient;
+    private SymConfig config;
+    private DatafeedEventsService datafeedEventsService;
+    private DatafeedClient datafeedClient;
+
+    @Before
+    public void initClient() {
+        symBotClient = mock(SymBotClient.class);
+        config = mock (SymConfig.class);
+        when(config.getDatafeedVersion()).thenReturn("v1");
+        when(symBotClient.getConfig()).thenReturn(config);
+
+        datafeedClient = mock(DatafeedClient.class);
+        when(symBotClient.getDatafeedClient()).thenReturn(datafeedClient);
+        when(datafeedClient.createDatafeed()).thenReturn("123");
+        when(datafeedClient.getAckId()).thenReturn("ack_id_string");
+    }
+
+    @SneakyThrows
+    @Test
+    public void readDatafeedTest() {
+        datafeedEventsService = new DatafeedEventsService(symBotClient);
+        // Wait for async operation to finish
+        Thread.sleep(1500);
```

Comment:
Is it possible to reduce the sleep duration? `1500` will impact our global tests execution time, what could be the minimal value?

Reply:
I reduced the duration to `500`